frontend/packages/operator-lifecycle-manager/mocks.ts

```diff
@@ -728,6 +755,9 @@ export const operatorHubListPageProps = {
     ] as PackageManifestKind[],
   },
   clusterServiceVersions: null,
+  cloudcredentials: { loaded: true, data: manualCred },
```

Comment:
If these are optional, you shouldn't need to update all of these.  Only the ones you want to test for the alert.